sdk/test/Services/DynamoDBv2/IntegrationTests/Config/35/App.config

```diff
@@ -36,6 +36,12 @@
             <property name="InternalId" ignore="true" />
             <property name="CurrentStatus" attribute="Status" converter="AWSSDK_DotNet.IntegrationTests.Tests.DynamoDB.DynamoDBTests+StatusConverter, AWSSDK.IntegrationTests.DynamoDBv2.Net35" />
           </map>
+          <map type="AWSSDK_DotNet.IntegrationTests.Tests.DynamoDB.DynamoDBTests+PartiallyAnnotatedEmployee, AWSSDK.IntegrationTests.DynamoDBv2.Net35" targetTable="HashRangeTable">
+            <property name="ManagerName" attribute="Manager" />
+            <property name="CompanyName" attribute="Company" />
+            <property name="InternalId" ignore="true" />
+            <property name="CurrentStatus" attribute="Status" converter="AWSSDK_DotNet.IntegrationTests.Tests.DynamoDB.DynamoDBTests+StatusConverter, AWSSDK.IntegrationTests.DynamoDBv2.Net45" />
```

Comment:
A lot of .NET Framework 3.5 tests are failing for me with the following, I think related to the `Net45` reference here.

```
Test method AWSSDK_DotNet.IntegrationTests.Tests.CacheTests.ChangingTableTest threw exception: 
System.TypeInitializationException: The type initializer for 'Amazon.AWSConfigsDynamoDB' threw an exception. ---> System.Configuration.ConfigurationErrorsException: The value of the property 'converter' cannot be parsed. The error is: The type 'AWSSDK_DotNet.IntegrationTests.Tests.DynamoDB.DynamoDBTests+StatusConverter, AWSSDK.IntegrationTests.DynamoDBv2.Net45' cannot be resolved. Please verify the spelling is correct or that the full type name is provided.
```

Reply:
Fixed